OpenSim/Common/Exception.cpp

```diff
@@ -70,8 +70,21 @@ exception()
 Exception::Exception(const std::string& file,
                      size_t line,
                      const std::string& func) {
-    addMessage(file + ":" + std::to_string(line) + "\n" + "In function '" + 
-               func + "'");
+    addMessage("\tIn file " + file + ":" + std::to_string(line) + "\n" +
+               "\tIn function '" + func + "'");
+}
+
+Exception::Exception(const std::string& file,
+              size_t line,
+              const std::string& func,
+              const std::string& className,
```

Comment:
What do you think of passing a const ref to the object itself? And strip the className and objectName out for us?